changelog/unreleased/kong/feat-request-debguger-finer-resolution-and-total-latency.yml

```diff
@@ -0,0 +1,6 @@
+message: |
+  Improve the latency resolution of the request debugger from milliseconds to microseconds when latency is lower than 1ms.
+  Add a new field `total_time_without_upstream` to request debugger output to show the total latency of the request without upstream latency.
```

Comment:
All thing into the `message` will be translated into a item of list in markdown during the release, so let's make it better.

For example

```markdown
message: |
  Improved the output of the request debugger
  - Now the resolution of field `total_time` is microseconds when it is less than 1ms instead of confusing 0ms.
  - A new field `total_time_without_upstream` on the top level shows the latency only introduced by Kong
```

Reply:
Thanks. Already fixed.